deployments/dev/config/common.yaml

```diff
@@ -69,6 +69,9 @@ jupyterhub:
         default: true
       - display_name: "Dashboard image"
         description: "A newer repo2docker-based image with a primary focus of building dashboards."
+      extraEnv:
```

Comment:
@balajialg It looks like the env is not indented the same as one of the profile stanzas. Also, I think `extraEnv` is not evaluated at the level of child of [profile_list](https://jupyterhub-kubespawner.readthedocs.io/en/latest/spawner.html#kubespawner.KubeSpawner.profile_list). It might have to be `extra_env` and as a child of `kubespawner_override`, if you wanted it to be specific to one of the profiles.

Reply:
Ah, got it. Thanks @ryanlovett. I will try to fix it in the next PR

Reply:
Yes, I wanted it to be the child of kubespawner_override